presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeTpcdsQueries.java

```diff
@@ -430,7 +430,8 @@ public void testTpcdsQ1()
         assertQuery(session, getTpcdsQuery("01"));
     }
 
-    @Test
+    // TODO(spershin): Enable test when fixed.
```

Comment:
Would you file GitHub issue with an error message and add a reference here?

CC: @frankobe @aditi-pandit @majetideepak

Reply:
Ugh, sure, that can be done.

Reply:
@spershin Let me verify by advancing velox locally, this query may be able to execute successfully now